plugins/web/opentelemetry-instrumentation-document-load/src/documentLoad.ts

```diff
@@ -256,22 +254,23 @@ export class DocumentLoad extends BasePlugin<unknown> {
     if (window.document.readyState === 'complete') {
       this._onDocumentLoaded();
     } else {
+      this._onDocumentLoaded = this._onDocumentLoaded.bind(this);
       window.addEventListener('load', this._onDocumentLoaded);
     }
   }
 
   /**
    * implements patch function
    */
-  protected patch() {
+  enable() {
+    this.disable();
```

Comment:
not sure why you are disabling the plugin when its enable, thats seems counter intuitive ? i think its worth to explain it in a comment